test/textAnalytics.js

```diff
@@ -65,4 +65,27 @@ describe('textAnalytics', () => {
         const result = client.preprocessText ("```abc\ndef\n```ghi");
         expect(result).to.be.equal("      \n   \n   ghi");
     });
+
+    it('Image tag 1', () => {
+        const text = client.preprocessText ("abc![image](abc)def");
```

Comment:
I edited my comment here: https://github.com/jonathanpeppers/inclusive-code-comments/issues/57#issuecomment-1012581822

![image](https://user-images.githubusercontent.com/840039/151576015-2866a670-2db8-4df9-91ed-8d2fcdd29058.png)

I don't really see what is different, the text is:
```
https://dataexplorer.azure.com/dashboards/a8390f81-19c6-4bf5-bbd6-1a84cf03d80c?_startTime=30days&_endTime=now#a32efffa-608f-4ad9-b685-4b414b5c305b

So the data we have as of now:

![image](https://user-images.githubusercontent.com/840039/149419837-d2f0f094-5aa5-4823-912c-c515e987b0c6.png)

LOL? 197 negative words/sentences yesterday?

![image](https://user-images.githubusercontent.com/840039/149419909-05fb80e4-cd3b-4351-98f5-220e07b36290.png)

This one seems reasonable that about 10 suggestions applied per day.

![image](https://user-images.githubusercontent.com/840039/149419971-f34a46f9-4b79-4eff-8bb8-6ee56f8bfd90.png)

Questionable if this is right? It should be a new user if you uninstall/reinstall the extension.

![image](https://user-images.githubusercontent.com/840039/149420085-8f9a9b6c-5bef-41a0-a536-9f1e1f7ad89a.png)

This one might also be right, if no one ignored anything.
```

Reply:
It looks like the negativity comes from the next line, you can see that the first character of the next line is underlined